ledger/ledger-api-auth/src/main/scala/com/digitalasset/ledger/api/auth/AuthServiceJWTPayload.scala

```diff
@@ -188,31 +188,45 @@ object AuthServiceJWTCodec {
       val scopes = scope.toList.collect({ case JsString(scope) => scope.split(" ") }).flatten
       // We're using this rather restrictive test to ensure we continue parsing all legacy sandbox tokens that
       // are in use before the 2.0 release; and thereby maintain full backwards compatibility.
-      val audienceValue = readOptionalStringOrSingletonArray(propAud, fields)
+      val audienceValue = readOptionalStringOrArray(propAud, fields)
```

Comment:
Can you extend the tests to cover the below additions, please?

Reply:
In the process of doing so.

Reply:
Done, fixed a small issue discovered by the tests as well.